model/src/test/scala/uk/gov/ons/addressIndex/model/AddressIndexModelTest.scala

```diff
@@ -7,34 +7,27 @@ class AddressIndexModelTest extends FlatSpec with Matchers {
 
   it should "produce the correct `AddressScheme` (`PostcodeAddressFile`) for the given string `paf`" in {
     val sample = "paf"
-    sample.toAddressScheme map (_ shouldBe a [PostcodeAddressFile])
+    sample.stringToScheme  shouldBe a [PostcodeAddressFile]
   }
 
   it should "produce the correct `AddressScheme` (`PostcodeAddressFile`) for the given string `postcodeAddressFile`" in {
     val sample = "postcodeAddressFile"
-    sample.toAddressScheme map (_ shouldBe a [PostcodeAddressFile])
+    sample.stringToScheme shouldBe a [PostcodeAddressFile]
   }
 
   it should "produce the correct `AddressScheme` (`BritishStandard7666`) for the given string `britishstandard7666`" in {
     val sample = "britishstandard7666"
-    sample.toAddressScheme map(_ shouldBe a [BritishStandard7666])
+    sample.stringToScheme shouldBe a [BritishStandard7666]
```

Comment:
Codacy is complaining that one should not use the name of the class (AddressScheme) in the method